### PR TITLE
Assessment condition satisfied with submitted submission.

### PR DESCRIPTION
**Pull request by custode**
Depends on #853

### CODE REVIEW

File: spec/models/course/condition/assessment_spec.rb [hunk 1]
```diff
@@ -93,14 +93,22 @@
           end
         end
 
-        context 'when the submission is not graded' do
-          it 'returns false' do
+        context 'when the submission is attempted' do
+          it 'return false' do
```
Comment (lowjoel) [inline spec/models/course/condition/assessment_spec.rb]: returns is correct

File: spec/models/course/condition/assessment_spec.rb [hunk 2]
```diff
@@ -117,34 +125,50 @@
...
-              answer.grade = 6
-              answer.save!
+        context 'when there are graded submissions' do
+          let(:submission) do
+            create(:submission, workflow_state: :graded, assessment: assessment,
+                                user: course_user.user)
+          end
+
+          context 'when there is no answer' do
+            it 'return false' do
```
Comment (lowjoel) [inline spec/models/course/condition/assessment_spec.rb]: returns

**Comment by lowjoel** [comment]
@custode can you rebase this please?

**Comment by custode** [comment]
@lowjoel Done rebasing.

**Pull request merged**